### PR TITLE
Bump to 3.4.x

### PR DESCRIPTION
**Pull request by nickpatrick**

### CODE REVIEW

File: example/ios/Example.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -303,7 +303,7 @@
 					"FB_SONARKIT_ENABLED=1",
 				);
 				INFOPLIST_FILE = example/Info.plist;
-				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.1;
```
Comment (jrtibbetts) [inline example/ios/Example.xcodeproj/project.pbxproj:306]: Interesting. Any particular reason? The native SDK still supports 10.

**Review by jrtibbetts — APPROVED** [review]
You didn't have to do anything with the `RadarRouteMode` declarations?

File: example/ios/Example.xcodeproj/project.pbxproj (hunk 1 above)
Comment (nickpatrick) [reply example/ios/Example.xcodeproj/project.pbxproj:306]: I was running into a build error that I ended up fixing by updating the Xcode version instead. May revert this.

Comment (nickpatrick) [reply example/ios/Example.xcodeproj/project.pbxproj:306]: > You didn't have to do anything with the `RadarRouteMode` declarations?

Not sure yet. Still trying to repro the original issue and then will make changes if needed.

Comment (nickpatrick) [reply example/ios/Example.xcodeproj/project.pbxproj:306]: Fixed this by moving `RadarRouteMode` to a separate header and bumping versions.

**Pull request merged**